src/app/components/Filters/FieldsetDate.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class FilterPopup extends React.Component {
+  render() {
+    return (
+      <fieldset>
+        <legend>Date</legend>
+        <div>
+          <label htmlFor="start-date">Start Year
+            <input id="start-date" name="start-date" className="form-text" type="date" />
+          </label>
+          <label htmlFor="end-date">End Year
+            <input id="end-date" type="date" className="form-text" name="end-date" />
+          </label>
+          <span>The Start year cannot be later than the end year</span>
+        </div>
+      </fieldset>
+    );
+  }
+}
+
+FilterPopup.propTypes = {
+  location: PropTypes.object,
```

Comment:
I don't see we have `this.props.location` here. We might not need the validation here.

Reply:
This is just a placeholder component that renders DOM. It doesn't work but I left that there so props can be added/removed when it gets updated.

Reply:
OK